inc/deployfile.class.php

```diff
@@ -53,6 +53,7 @@ class PluginGlpiinventoryDeployFile extends PluginGlpiinventoryDeployPackageItem
     const REGISTRY_NO_DB_ENTRY = 0x1;
     const REGISTRY_NO_MANIFEST = 0x2;
 
+    const DL_MAX_SIZE = 1024 * 1024 * 50;
```

Comment:
Why would we have a download size limit?

Reply:
As far as I understand, file is generated, and written on disk; whatever its size is

Reply:
You can close `session_write_close()` to prevent blocking session.